cmd/server/assets/realmadmin/_stats_keyserver.html

```diff
@@ -402,7 +402,8 @@ <h5 class="modal-title">Onset-to-upload distribution</h5>
       hAxis: {
         title: 'TEK days old',
         gridlines: { color: 'transparent' },
-        ticks: [{v:1, f:'<1'},2,3,4,5,6,7,8,9,10,11,12,13,14,15],
+        baseline: { color: 'transparent' },
+        ticks: [0,1,2,3,4,5,6,7,8,9,10,11,12,13,14],
```

Comment:
last as >=14

Reply:
the last column is left off the ticks and instead gets a label and a different color